academic_affairs.sql

```diff
@@ -16,11 +16,23 @@ WITH gather_data AS (
         grade_c = '12th Grade'
         OR (
           grade_c = 'Year 1'
-          AND years_since_hs_grad_c = 0
+          AND indicator_years_since_hs_graduation_c = 0
         )
       ) THEN 1
       ELSE 0
-    END AS composite_ready
+    END AS composite_ready,
+    CASE
```

Comment:
@jbakerr , what kpi is this bit for? think I just need to understand intention

Reply:
Same, think some grounding here would be good!

Reply:
@jvolan123 and @isabelreyna if you click the expand icon in the left side bar (the one with row numbers) you can see the code above this where the KPI is pointed out. In this case it is 
    -- % of seniors who are composite ready

Reply:
**% of seniors who are composite ready** would apply to the CASE statement above row 25, no? I think our questions are specific to rows 25 - 35 (at least mine are)

Reply:
oh sorry, you're right this applies to the test growth KPI - % of not ready and near ready students who become composite ready by the highest official exam score

my bad! I'm creating the denominator here in the case statement.